src/main.rs

```diff
@@ -27,19 +27,15 @@ struct Cat{
     status: String,
 }
 
-
 fn main() -> Result<(), Box<std::error::Error>> {
-    let mut resp = reqwest::get("https://ra-api.adoptapet.com/v1/shelters/75215/pets/?start=1&limit=100")?.text()?;
+    let mut resp =
+        reqwest::get("https://ra-api.adoptapet.com/v1/shelters/75215/pets/?start=1&limit=100")?
+            .text()?;
     let cats: Value = serde_json::from_str(&resp)?;
```

Comment:
why do we need this

Reply:
I don't understand my b's question. from_str takes a raw web request response and converts it into a json object. We want the json object so that we can access the various fields of the response more easily. Does that answer my b's question? <3

Reply:
line 35 is being duplicated, looks like this is jsut for printing..

Reply:
some people just want to watch the world 🔥